core/messages/de.json

```diff
@@ -0,0 +1,329 @@
+{
+    "RootLayout": {
+        "Metadata": {
+            "title": "Catalyst Store",
+            "description": "Example store built with Catalyst",
+            "other": {
+                "platform": "bigcommerce.catalyst"
+            }
+        }
+    },
+    "Home": {
+        "Carousel": {
+            "bestSellingProducts": "Best Selling Products",
+            "featuredProducts": "Empfohlene Produkte",
+            "newestProducts": "Newest products"
+        }
+    },
+    "Category": {
+        "sortBy": "{items, plural, =0 {no items} =1 {1 item} other {# items}}",
+        "products": "Produkte",
+        "items": "Artikel"
+    },
+    "Brand": {
+        "sortBy": "{items, plural, =0 {no items} =1 {1 item} other {# items}}",
+        "products": "Produkte"
+    },
+    "Product": {
+        "loading": "Wird geladen …",
+        "carouselTitle": "Verwandte Produkte",
+        "noGalleryText": "In Kürze verfügbar",
+        "DescriptionAndReviews": {
+            "description": "Beschreibung",
+            "warranty": "Garantie",
+            "loading": "Wird geladen …",
+            "reviews": "Bewertungen",
+            "reviewsCount": "Count:",
+            "unreviewed": "This product hasn't been reviewed yet.",
```

Comment:
I see that we miss some translations across phrases like here. Is it expected?

Reply:
Yes, not all phrases are translated already, more will come in future updates.